lib/greenbar/engine.ex

```diff
@@ -41,15 +41,36 @@ defmodule Greenbar.Engine do
     end
   end
 
-  def eval!(%__MODULE__{}=engine, name, scope \\ %{}) when is_binary(name) do
+  def eval!(%__MODULE__{}=engine, name, opts) when is_binary(name) do
+    scope = Keyword.get(opts, :scope, %{})
     case get_template(engine, name) do
       nil ->
         {:error, :not_found}
       template ->
-        Template.eval!(template, engine, scope)
+        directives = Template.eval!(template, engine, scope)
+        case select_renderer(Keyword.get(opts, :render)) do
+          nil ->
+            directives
+          render_mod ->
+            directives
+            |> Poison.encode!
+            |> Poison.decode!
```

Comment:
I guess we're doing this to convert the map atom syntax to the string syntax. Would it better to just switch the renderers over to symbols? I don't mind doing that if'n you think it would be better.